src/_pytest/python.py

```diff
@@ -525,18 +583,19 @@ def _importtestmodule(self):
         self.config.pluginmanager.consider_module(mod)
         return mod
 
-    def setup(self):
-        setup_module = _get_xunit_setup_teardown(self.obj, "setUpModule")
-        if setup_module is None:
-            setup_module = _get_xunit_setup_teardown(self.obj, "setup_module")
-        if setup_module is not None:
-            setup_module()
-
-        teardown_module = _get_xunit_setup_teardown(self.obj, "tearDownModule")
-        if teardown_module is None:
-            teardown_module = _get_xunit_setup_teardown(self.obj, "teardown_module")
-        if teardown_module is not None:
-            self.addfinalizer(teardown_module)
+    # def setup(self):
```

Comment:
please remove the comments, dead code should go, we have history so we don't have to keep zombies :smiling_imp:

Reply:
Oh definitely. 👍